frontend/src/App.js

```diff
@@ -211,6 +211,13 @@ export default function App() {
                   component={() => <Home />}
                   role=""
                 />
+                <SecureRoute
```

Comment:
you dont need this Route . 
the existing Route for `/RoutineReport` alrready handle your report